### PR TITLE
fix: use proper custom scope when being passed via params

### PR DESCRIPTION
**Pull request by marckraw**

### CODE REVIEW

File: src/storyblok-auth-api/grant/grant-handler.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export const grantHandler =
         storyblok: {
           client_id: clientId,
           client_secret: clientSecret,
-          scope: ['read_content'],
+          scope: scope ? scope : ['read_content'],
```
Comment (johannes-lindgren) [inline src/storyblok-auth-api/grant/grant-handler.ts:35]:
```suggestion
          scope,
```

The scope is a required parameter, so no need to use a fallback value

**Review by johannes-lindgren — CHANGES_REQUESTED** [review]

**Comment by marckraw** [comment]
@johannes-lindgren should I have access to this endpoint though ? (its not on your list), can i find a list of available endpoints somewhere ? 

```
spaces/${spaceId}/presets/${p.preset.id}
```

**Comment by marckraw** [comment]
ah ok i just saw, multiple presets doesnt work..

**Comment by chrsbrmr** [comment]
are there any updates on this? since my accessToken isn´t authorized for anything.

**Comment by marckraw** [comment]
@johannes-lindgren yes, is there a way to see what are the status of the exposing API endpoints ?

**Pull request closed without merging**